app/src/main/java/edu/uc/mille5bm/smartnutrition/dto/Food.kt

```diff
@@ -1,4 +1,4 @@
-/** Holds information about Food ..*/
```

Comment:
- Use proper JavaDoc syntax.
- The statement here is redundant, given that the class name is Food.  If adding JavaDoc, provide information that cannot be determined by reading the class name alone.